samples/qa-a11y/src/views/ReadAlert.js

```diff
@@ -21,56 +18,50 @@ class ReadAlertView extends Component {
 					keys: ['audioGuidance']
 				},
 				onSuccess: (res) => {
-					this.setState({
-						audioGuidance: res.settings.audioGuidance === 'on',
-						toggleDisabled: false
-					});
+					setAudioGuidance(res.settings.audioGuidance === 'on');
+					setToggleDisabled(false);
 				}
 			});
 		}
+	});
+
+	const onClick = (clear) => () => readAlert('Enact is a framework designed to be performant, customizable and well structured.', clear);
 
-		this.onClick1 = this.onClick(true);
-		this.onClick2 = this.onClick(false);
-	}
 
-	onClick = (clear) => () => readAlert('Enact is a framework designed to be performant, customizable and well structured.', clear);
+	const onClick1 = useCallback(() => onClick(true), []);
+	const onClick2 = useCallback(() => onClick(false), []);
```

Comment:
The onClick method was not called with your implementation. here is a suggestion
```suggestion
	const onClick1 = onClick(true);
	const onClick2 = onClick(false);
```